src/main/java/com/checkmarx/flow/service/EmailService.java

```diff
@@ -39,34 +46,36 @@ public EmailService(FlowProperties properties, TemplateEngine templateEngine, Ja
      * @param ctx
      * @param template
      */
-    public void sendmail(List<String> recipients, @NotNull String subject, @NotNull Map<String, Object> ctx, String template){
-        if(ScanUtils.empty(recipients)){
-            log.info("Email not sent - no recipients listed");
-            return;
-        }
+    public void sendmail(List<String> recipients, @NotNull String subject, @NotNull Map<String, Object> ctx, String template) {
+
         MimeMessagePreparator messagePreparator = mimeMessage -> {
-            log.info("Sending email");
+            log.info("Sending email notification.");
             MimeMessageHelper messageHelper = new MimeMessageHelper(mimeMessage);
 
-            String contact = properties.getContact();
-            if(!ScanUtils.empty(contact)){
+            String contact = flowProperties.getContact();
+            if (!ScanUtils.empty(contact)) {
                 messageHelper.setFrom(contact);
             }
-            messageHelper.setTo(recipients.toArray(new String[0]));
+
+            if (!ScanUtils.empty(recipients)) {
+                messageHelper.setTo(recipients.toArray(new String[0]));
+                messageHelper.setCc((flowProperties.getMail().getCc()).toArray(new String[0]));
+            } else {
+                messageHelper.setTo((flowProperties.getMail().getCc()).toArray(new String[0]));
+            }
+
             messageHelper.setSubject(subject);
             String content = generateContent(ctx, template);
             messageHelper.setText(content, true);
         };
         try {
-            FlowProperties.Mail mail = properties.getMail();
-            if(mail != null && mail.isEnabled()) {
+            FlowProperties.Mail mail = flowProperties.getMail();
+            if (mail != null) {
```

Comment:
Why was isEnabled removed from the condition?

Reply:
isEnabled was removed to keep the functionality separate for usual PR notification mails and Email as a bug tracker. 
Appropriate conditions are used at the point where the mails are triggered. So this condition was being redundant.